spock-container-test-app/build.gradle

```diff
@@ -1,67 +1,66 @@
-plugins {
-        id "groovy"
-        id "org.grails.grails-web"
-        id "org.grails.grails-gsp"
-        id "war"
-        id "idea"
-        id "com.bertramlabs.asset-pipeline"
-        id "eclipse"
+buildscript {
+    repositories {
+        maven { url = 'https://repo.grails.org/grails/core/' }
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
+        classpath "org.grails:grails-gradle-plugin:$grailsVersion"
+    }
 }
 
-group = "org.demo.spock"
+apply plugin: 'groovy'
```

Comment:
Why switch from the plugin{} syntax?

Reply:
```console
* What went wrong:
Plugin [id: 'org.grails.grails-web'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (plugin dependency must include a version number for this source)
```

Reply:
Isn't that because you don't have something like this in the settings gradle:

```
pluginManagement {
    repositories {
        mavenLocal()
        maven { url "https://repo.grails.org/grails/core/" }
        gradlePluginPortal()
    }
    plugins {
        id "org.grails.grails-web" version "${grailsGradlePluginVersion}"
        id "org.grails.grails-gsp" version "${grailsGradlePluginVersion}"
        id "org.grails.grails-plugin" version "${grailsGradlePluginVersion}"
        id "org.grails.grails-publish" version "${grailsGradlePluginVersion}"
    }
}
```

Reply:
Yes